fullstack/env/development.shorturl.env

```diff
@@ -0,0 +1,23 @@
+NODE_ENV=development
+
+PORT=3333
+
+BASE_URL=http://localhost:3332
+
+DATABASE_NAME=default
+DATABASE_TYPE=mysql
+DATABASE_HOST=127.0.0.1
+DATABASE_PORT=3306
+DATABASE_USER=root
+DATABASE_DB=short_url_db
+DATABASE_PWD=123456
+DATABASE_SYNCHRONIZE=true
+DATABASE_DROPSCHEMA=true
+
+# token config
+TOKEN_EXPIRES_IN=1d
+TOKEN_EXPIRES_IN_S=604800s
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: High

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_ignore_everywhere** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_revoked** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

 <!-- b2af787d-b73b-41fb-9dfa-214c34ccec8c -->